### PR TITLE
profile page

### PR DESCRIPTION
**Pull request by ChitvanRamani22**
## Profile Page

Closes: #140 

Screenshot (488)
Screenshot (489)
@codewithvk please review the pr.

### CODE REVIEW

**Review by codewithvk — APPROVED** [review]
LGFM :).@purnima143 Any suggestion for design?

**Comment by ChitvanRamani22** [comment]
@codewithvk design was given by @purnima143 in the issue.
Further i could do if something is required.

File: client/src/components/profilePage/profilePage.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                        <span>|</span>
+                        <Link to="/addaquestion">Questions</Link>
+                        <span>|</span> <Link to="feed">Answers</Link>
+                    </h3>
+                    <hr></hr>
+                </div>
+                <div className="followers">
+                    <h3>
+                        {" "}
+                        Following - <span>10</span> &nbsp; &nbsp;
```
Comment (codewithvk) [inline client/src/components/profilePage/profilePage.js:39]: One suggestion! Make this static value of followers convert into a variable and declare above.

Comment (ChitvanRamani22) [reply client/src/components/profilePage/profilePage.js:39]: According to github followers?

Comment (codewithvk) [reply client/src/components/profilePage/profilePage.js:39]: No no, Just declare variable in above, so when data comes from backend than that would become easy to find.
ex. `const following = 0`
`Following - <span>{following}</span> &nbsp; &nbsp;`

Comment (ChitvanRamani22) [reply client/src/components/profilePage/profilePage.js:39]: Okay fine both both followers and following?

**Pull request merged**